### PR TITLE
The old test for the expectation not_to_be_null refactored

### PR DESCRIPTION
**Pull request by AlexisGaldamez**
I refactored the old test for the expectation not_to_be_null and delete the old test

### CODE REVIEW

File: test/ut_expectations/test_expec_not_to_be_null.pkb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    end;
+
+    procedure test_success_expectacion(a_data_type in varchar2,
+                                    a_data_value in varchar2)
+    is
+    begin
+        execute_expectation(a_data_type, a_data_value);
+
+        --assert - check that expectation was executed successfully
+        ut3_latest_release.ut.expect(ut3.ut_expectation_processor.get_status()).to_equal(ut3.ut_utils.tr_success);
```
Comment (jgebal) [inline test/ut_expectations/test_expec_not_to_be_null.pkb]: Please remove reference to `ut3_latest_release`
We use utPLSQL v3 (latest release) to test our software (utPLSQL in ut3 schema).
In  **development mode** we do not have synonyms for `ut3` schema.
The synonyms are created to the `ut3_latest_release`

File: test/ut_expectations/test_expec_not_to_be_null.pks [hunk 2]
```diff
@@ -0,0 +1,65 @@
+create or replace package test_expec_not_to_be_null
+is
+    --%suite(expectations - not_to_ve_null)
```
Comment (jgebal) [inline test/ut_expectations/test_expec_not_to_be_null.pks]: Can you fix the typo?

File: test/ut_expectations/test_expec_not_to_be_null.pks [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    --%suitepath(utplsql.core.expectations.no_to_be_null)
+
+    --%test(test blob not null)
```
Comment (jgebal) [inline test/ut_expectations/test_expec_not_to_be_null.pks]: No need to put "test" in the test description. We all know it's a test.
Best way to name tests is by describing behavior that they test. This way you have feature documentation straight from your tests.
Example:
`--%test(Gives success for a not null BLOB)

File: test/ut_expectations/test_expec_not_to_be_null.pks [hunk 4]
```diff
@@ -0,0 +1,65 @@
+create or replace package test_expec_not_to_be_null
+is
+    --%suite(expectations - not_to_ve_null)
+    --%suitepath(utplsql.core.expectations.no_to_be_null)
```
Comment (jgebal) [inline test/ut_expectations/test_expec_not_to_be_null.pks]: Can you fix the typo in suite path

File: test/ut_expectations/test_expec_not_to_be_null.pkb [hunk 5]
```diff
@@ -0,0 +1,175 @@
+create or replace package body test_expec_not_to_be_null
```
Comment (jgebal) [inline test/ut_expectations/test_expec_not_to_be_null.pkb]: I think the package should be named:
 test_expec**t**_not_to_be_null

File: old_tests/ut_expectations/ut.expect.not_to_be_null.sql [hunk 6]
```diff
@@ -1,19 +0,0 @@
-@@lib/RunTest.sql "ut_expectations/common/ut.expect.unuary_comparator.scalar.common.sql 'blob' 'to_blob(''abc'')' 'not_to_be_null' 'ut_utils.tr_success'"
```
Comment (jgebal) [inline old_tests/ut_expectations/ut.expect.not_to_be_null.sql]: Should we also remove `ut.expect.unuary_comparator.scalar.common.sql` or is it still used by other tests?

Comment (AlexisGaldamez) [reply old_tests/ut_expectations/ut.expect.not_to_be_null.sql]: We  can't do it, It's still used for two test

**Pull request merged**